### DISCUSSION TITLE
How to use the Icons Kit (FontAwesome Pro) using NPM and not the FontAwesome CDN?

### DISCUSSION THREAD

**Discussion by GitHubish** [FortAwesome/Font-Awesome#19021] [Q&A] [answered]
I need the custom icons I add to my Kit to be usable with [Angular](https://github.com/FortAwesome/angular-fontawesome) without going through the CDN for an offline web application. Is this possible? I don't see any mention of this usage on the "How to : Add Your Kit's Code to a Project". 

Thanks

**Comment by sensibleworld** [comment]
@GitHubish This is not currently possible, but we're working on lots of improvements to Kits, and are hoping to add package managers as an option.

**Comment by YannFFT** [comment]
Hello,
Any update on this topic ?
I'd also like to use Kits with package managers.

**Reply by sensibleworld** [reply]
With the addition of custom icons and custom subsetting to Kits, this is more important to us than ever. This is currently in active development, and should be released in the next couple of months.

**Comment by sensibleworld** [comment]
Just a note that this feature is in active development. Stay tuned!

**Comment by sheadawson** [comment]
Hey, How's this going?

**Comment by sensibleworld** [comment, marked as answer]
@sheadawson Thanks for checking! Installing a Kit via NPM is now possible: https://docs.fontawesome.com/web/setup/packages
